assets/data/medialist.json

```diff
@@ -1,294 +1,294 @@
 [
   {
-    "author_name": "Carolyn Said",
```

Comment:
Should this be "id" or something?

Reply:
Good point, I just updated this column in Airtable to be `id`.  I don't believe the python file needs to be updated to reflect this as this primary key column is automatically included from what I can tell.